### PR TITLE
feat: add github workflows

### PR DESCRIPTION
**Pull request by friendtocephalopods**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  docs – A documentation improvement task
+  build – A task related to our build system
+  ci – A task related to our ci system
+  perf – A performance improvement
+  refactor – A code refactor PR
+  style – A task about styling
+  test – A PR that improve test coverage
+  chore – A regular maintenance chore or task
+  other – Any other kind of PR
+-->
```
Comment (worldwise001) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: Any reason why these are commented out?

File: .github/workflows/pypipublish.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  build-and-publish-python-module:
+    name: Build and publish python module to pypi
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup python 3.6
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
```
Comment (worldwise001) [inline .github/workflows/pypipublish.yml]: I feel like there's some inconsistencies of versions here. Like I see ubuntu 16 and 18, and python 3.6 and 3.7. Any way we can standardize those?

File: .github/tests.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+on: [push]
+
+jobs:
+  unit:
+    name: unit
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v1
+      with:
+        python-version: 3.7
```
Comment (worldwise001) [inline .github/tests.yml]: Do you want to allow for tests in 3.6 and 3.7?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (friendtocephalopods) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: It's the same format as the other repos, you'd normally delete after naming your PR appropriately I think

File: .github/tests.yml (hunk 3 above)
Comment (friendtocephalopods) [reply .github/tests.yml]: Hmm actually I think this one is replaced by the workflow below. I'll try removing, which will also stabilize ubuntu versions.

File: .github/workflows/pypipublish.yml (hunk 2 above)
Comment (friendtocephalopods) [reply .github/workflows/pypipublish.yml]: Hmm yeah I'll use 3.7 here.

**Comment by alran** [comment]
this is looking good to me! ty for doing this!

**Review by alran — APPROVED** [review]

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+# Reference: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# Each line is a file pattern followed by one or more owners.
+# Order is important; the last matching pattern takes the most
+# precedence.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @amundsen-io/amundsen-committerswill be requested for
+# review when someone opens a pull request.
+*       @amundsen-io/amundsen-committers
```
Comment (feng-tao) [inline .github/CODEOWNERS:11]: it could be @amundsen-io/amundsen-gremlin-committers as well.

Comment (friendtocephalopods) [reply .github/CODEOWNERS:11]: Oh, good idea!

**Pull request merged**